os-certmonger.te

```diff
@@ -0,0 +1,10 @@
+policy_module(os-certmonger,0.1)
+
+gen_require(`
+  type certmonger_t;
+  type puppet_etc_t;
+  class dir {search};
```

Comment:
As mentioned, the 'class' line isn't necessary either but it doesn't break anything either so let's merge this.